plugins/generator-1.20.1/forge-1.20.1/utils/triggers.java.ftl

```diff
@@ -1,6 +1,25 @@
 <#include "procedures.java.ftl">
 
 <#-- Item-related triggers -->
+<#macro addSpecialInformation procedure="" isBlock=false>
+    <#if procedure?has_content || hasProcedure(procedure)>
+		@Override public void appendHoverText(ItemStack itemstack, <#if isBlock>BlockGetter<#else>Level</#if> world, List<Component> list, TooltipFlag flag) {
+		super.appendHoverText(itemstack, world, list, flag);
+        <#if hasProcedure(procedure)>
+			Entity entity = itemstack.getEntityRepresentation();
+			double x = entity != null ? entity.getX() : 0.0;
+			double y = entity != null ? entity.getY() : 0.0;
+			double z = entity != null ? entity.getZ() : 0.0;
+			list.add(Component.literal(<@procedureOBJToStringCode procedure/>));
+        <#else>
+            <#list thelper.splitCommaSeparatedStringListWithEscapes(procedure.getFixedValue()) as entry>

```

Comment:
@Defeatomizer, is this ok, given now procedure.getFixedValue() returns list, I don't think splitCommaSeparatedStringListWithEscapes is actually needed.

![image](https://github.com/MCreator/MCreator/assets/16374228/fd9789d1-e74c-42dc-b6b3-6cc94bb9aa93)

Indentations also look a bit wild

Reply:
Should it be removed in both `TemplateHelper` and `StringUtils`?

Reply:
I would say so

Reply:
Done!